elfi/methods/bo/acquisition.py

```diff
@@ -547,6 +549,172 @@ def _evaluate_logpdf(theta):
         return batch_theta
 
 
+class ExpIntVar(MaxVar):
+    r"""The Expected Integrated Variance (ExpIntVar) acquisition method.
+
+    Essentially, we define a loss function that measures the overall uncertainty
+    in the unnormalised ABC posterior over the parameter space.
+    The value of the loss function depends on the next simulation and thus
+    the next evaluation location \theta^* is chosen to minimise the expected loss.
+
+    \theta_{t+1} = arg min_{\theta^* \in \Theta} L_{1:t}(\theta^*), where
+
+    \Theta is the parameter space, and L is the expected loss function approximated as follows:
+
+    L_{1:t}(\theta^*) \approx 2 * \sum_{i=1}^s (\omega^i * p^2(\theta^i)
+                                * w_{1:t+1})(theta^i, \theta^*), where
+
+    \omega^i is an importance weight,
+    p^2(\theta^i) is the prior squared, and
+    w_{1:t+1})(theta^i, \theta^*) is the expected variance of the unnormalised ABC posterior
+    at \theta^i after running the simulation model with parameter \theta^*
+
+    References
+    ----------
+    [1] arXiv:1704.00520 (Järvenpää et al., 2017)
+
+    """
+
+    def __init__(self, quantile_eps=.01, n_imp_samples=100, iter_imp=2,
+                 sampler='metropolis', n_samples=10000, sigma_proposals_metropolis=np.ones(1),
+                 *args, **opts):
```

Comment:
For expintvar, we use metropolis as the default because our 100 importance samples are taken from 10000 metropolis samples. Whereas 10000 metropolis samples for 1 acquisition point in randmaxvar would be an overkill (instead, we run NUTS with 50 samples).

Reply:
ok, but as discussed last week, it is not only about ESS but the reliability and correctness of the samples and I would think in that NUTS will be better generally (this is difficult to measure though). I can also imagine that is some other cases (especially if the proposal for MH is set badly) NUTS might be better as well as in higher dimensions

Reply:
Keep in mind that these are the default settings, the user could always choose her/his preference. What would be your suggestion for the chain length if we used NUTS?

Reply:
a bit hard to say as I am not too familiar with NUTS, maybe something like 500 might serve as a default? It is of course true that this is only the default and the user could (or should) choose his/her settings

Reply:
STAN's default is 4 chains, each with 1000 for warmup and 1000 for actual sampling, i.e. 8000 altogether. :D

Reply:
would that we a bit too costly here? maybe 500+500 (and only one chain) here as high accuracy is not very important

Reply:
Added an option to choose between a grid of points and a list of importance samples, and set the grid to be the default method with `1000` points in total.